templates/order/summary.txt.twig

```diff
@@ -1,4 +1,4 @@
-{{ 'order.summary.heading'|trans({ '%number%': order.number, '%id%': order.id }) }}
+{{ 'order.summary.heading'|trans({ '%number%': order.number, '%id%': order.id , '%tipAmount%' order.tipAmount}) }}
```

Comment:
Actually, it should be `order.getAdjustmentsTotal('tip')`, instead of `order.tipAmount`

Reply:
You also need to modify the translation [order.summary.heading](https://github.com/coopcycle/coopcycle-web/blob/40d422d861c1d1c91a204003d1dec9652941c441/translations/messages.en.yml#L292) to show the tip amount